public/js/clients/chrome.js

```diff
@@ -61,23 +61,7 @@ function onConnection(connection) {
 }
 
 function debugTab(tab) {
-  function toTitleCase() {
-    return this.substring(0, 1).toUpperCase() + this.substring(1);
-  }
-
-  // NOTE: this is a serious performance issue.
-  // If we decide to productionize the chrome debugger,
-  // we will want to build the bootstrap commands in advance require them in.
-
-  // NOTE: toTitleCase is temporarily monkey patched onto String.prototype
-  // to conform to what generateCommands expects. I
-  // If String.prototype.toTitleCase is defined somwhere else,
-  // it will be replaced.
-  String.prototype.toTitleCase = toTitleCase; /* eslint no-extend-native: 0 */
-  let code = generateCommands(protocol);
-  delete String.prototype.toTitleCase;
-  eval(code); /* eslint no-eval: 0 */
-
+  bootstrap(InspectorBackend);
```

Comment:
This comes from the file `bootstrap.js` though, which is built by `public/js/clients/chrome/build-bootstrap.js`. Are you supposed to run that file manually to update `bootstrap.js`, and commit the changes?